DEVNOTES.md

```diff
@@ -14,26 +14,28 @@
 ***
 
 
-## Building Ignite
-Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project root directory (you can disable the tests when building using `-x test`):
+## Quick Start
+Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project
```

Comment:
1. We should mention Apache.
2. Avoid `using`, its always hard to figure out. Change it to by using etc.
3. Avoid easily etc.
3. End sentence with dots or other punctuation.
Here's a suggestion:
Apache Ignite 3 follows standard guidelines for multi-module Gradle projects, so it can be built by using the following command from the project:

Reply:
Sounds good, will do it. It was the initial text though, I didn't change it. BTW, the sentence is continued on the next line.